src/snowflake/snowpark/functions.py

```diff
@@ -778,6 +778,50 @@ def convert_limit_to_col(limit):
     return builtin("uniform")(min_col, max_col, gen_col)
 
 
+def seq1(sign: int = 0) -> Column:
```

Comment:
`sign` needs an explanation

Reply:
Plus, add test coverage to these new functions.

Reply:
I just added coverage for all `seq` functions. I will update docstrings as you suggested